apps/test/unit/templates/instructions/ReviewTabTest.jsx

```diff
@@ -130,45 +146,62 @@ describe('Code Review Tab', () => {
 
       it('hides comment box if there is an authorization error on comment', () => {
         // first, enable peer review
-        stubReviewableStatusProjectServerCall({
+        stubReviewableStatusProjectApiCall({
           canMarkReviewable: false,
           reviewEnabled: true
         });
 
         const testCommentText = 'test comment text';
-        server.respond();
 
         // initial server response loads a comment
         expect(wrapper.find(CommentEditor).length).to.equal(1);
         expect(wrapper.find(Comment).length).to.equal(1);
 
         // fake a 404 for saving a comment
-        server.respondWith('POST', '/code_review_comments', [
-          404,
-          {'Content-Type': 'application/json'},
-          'error'
-        ]);
+        stubSubmitCommentError({status: 404});
 
         wrapper.instance().onNewCommentSubmit(testCommentText);
-        server.respond();
         // should still only have 1 comment, not 2
         expect(wrapper.find(Comment).length).to.equal(1);
         expect(wrapper.find(CommentEditor).length).to.equal(0);
         expect(wrapper.state().authorizationError).to.be.true;
       });
 
-      it('removes a comment when one is deleted', () => {
-        server.respond();
+      it('displays error message if comment contains profanity', () => {
+        // first, enable peer review
+        stubReviewableStatusProjectApiCall({
```

Comment:
True, this will produce the same result that the stubbing in `beforeEach` does, but I think i'll opt to leave it here to make it clear what expectations the test is setting up?